code/datums/ai/basic_mobs/targetting_datums/basic_targetting_datum.dm

```diff
@@ -17,8 +17,10 @@
 	var/check_factions_exactly = FALSE
 	/// Minimum status to attack living beings
 	var/stat_attack = CONSCIOUS
-	///Whether we care for seeing the target or not
+	/// Whether we care for seeing the target or not
 	var/ignore_sight = FALSE
+	/// Whether to target ONLY wounded targets
+	var/wounded_only = FALSE
```

Comment:
I don't know if I love this being a flag on all instances of this behaviour, maybe we need a `check_valid_living` proc you can extend or something. 
Frankly a few of these should be on the blackboard too, ideally don't want to start proliferating subtypes with inheritance largely based on different combinations of booleans.